src/main/java/duke/finalObjects/TootieErrorMsgs.java

```diff
@@ -0,0 +1,23 @@
+package duke.finalObjects;
+
+public class TootieErrorMsgs {
+    public static final String NEWLINE = System.lineSeparator();
+
+    public static final String TasklistEmptyMsg = "No tasks found! " + TootieSymbols.HAPPY_EMOTICON;
```

Comment:
is this a constant? If yes then consider changing to all caps? maybe you missed this, since the next few lines you did correctly use caps for constants